addons/website_slides_survey/models/survey_user.py

```diff
@@ -21,7 +22,8 @@ def create(self, vals_list):
     @api.multi
     def write(self, vals):
         res = super(SurveyUserInput, self).write(vals)
-        self._check_for_failed_attempt()
+        if 'state' in vals or 'quizz_passed' in vals or 'slide_partner_id' in vals:
```

Comment:
`quizz_passed` is a computed field. We will never see it in the write values.
Why checking slide_partner_id ?

Reply:
I changed it to "done" only.
This is the only field that's relevant because we want to check only when the user_input is actually done.